src/nats.h

```diff
@@ -2878,6 +2878,26 @@ natsConnection_GetRTT(natsConnection *nc, int64_t *rtt);
 NATS_EXTERN natsStatus
 natsConnection_HasHeaderSupport(natsConnection *nc);
 
+/** \brief Returns the connection local IP and port.
+ *
+ * Unlike #natsConnection_GetClientIP, this function returns the
```

Comment:
Might be helpful to point to this option in the natsConnection_GetClientIP documentation

Reply:
Will do.